pluto_build/bin/config.sh

```diff
@@ -92,8 +92,9 @@ function imports_csv {
 }
 
 function Upload {
-  mc rm -r --force spaces/edm-publishing/db-pluto/$@
-  mc cp -r output spaces/edm-publishing/db-pluto/$@
+  local target_folder=$1
+  mc cp -r output spaces/edm-publishing/db-pluto/$target_folder/$DATE
+  mc cp -r output spaces/edm-publishing/db-pluto/$target_folder/latest
```

Comment:
I see this now follows the convention we discussed in standup today. Looks good to me!

Reply:
te can I get an approval of the PR itself so we can merge in?

Reply:
Hold on still trying to test everything. Will give you a heads up when I finished. Thanks!

Reply:
ah ok great, sorry didn't realize you were still working